rules/linux/credential_access_credential_dumping.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2023/02/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/02/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of the Linux utility 'unshadow' being executed with the right set of parameters.
+This technique could be used by malicious actors to retrieve the contents of the '/etc/shadow' and
+'/etc/password' files.Using these files the malicious threat actors can prepare
+themselves for password-cracking utilities.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Linux Credential Dumping"
```

Comment:
Maybe we should be more specific here by specifying the utility we are detecting and maybe use cracking preparation instead of dumping, although it also makes sense to use dumping here

Reply:
Like i said, the utility can be absued for Reconnaissance purpose as well and we will introduce name of the utility and be very specific to credential dumping.

Reply:
https://github.com/elastic/detection-rules/pull/2604#discussion_r1119989898